src/Pages/LoginPage.ts

```diff
@@ -0,0 +1,125 @@
+import { type PageInterface } from "@odg/chemical-x";
+import { inject, injectable } from "inversify";
+import { expect } from "playwright/test";
+
+import Container from "@app/Container";
+import { ConfigName, ContainerName } from "@app/Enums";
+import { BasePage } from "@pages/BasePage";
+import {
+    type LoginSelectorType,
+    loginSelector,
+} from "@selectors";
+
+/**
+ * Generated page Login
+ *
+ * @author Leonardo Dollinger <leonardo.dollinger_iebt@123milhas.com.br>
+ */
+@injectable()
+export class LoginPage extends BasePage implements PageInterface {
+
+    @inject(ContainerName.Container)
+    public readonly container!: Container;
+
+    /**
+     * Selector to LoginPage
+     *
+     * @type {LoginSelectorType}
+     */
+    public $s: LoginSelectorType = loginSelector;
+
+    public hasLogged: boolean = false;
+
+    /**
+     * Execute this step
+     *
+     * @returns {Promise<void>}
+     */
+    public async execute(): Promise<void> {
+        await this.start(async () => {
+            await this.preStart();
+            await this.homeStep();
+            await this.loginStep();
+        });
+    }
+
+    /**
+     * Process home
+     *
+     * @returns {Promise<void>}
+     */
+    public async homeStep(): Promise<void> {
+        // eslint-disable-next-line unicorn/numeric-separators-style
+        await this.page.goto("https://www.latamairlines.com/br/pt", { timeout: 15000 });
+        try {
+            await expect(this.page.locator(this.$s.cookiesBtn)).toBeVisible({ timeout: 5000 });
```

Comment:
expect é usado para testes, nesse caso é esperado apenas exception caso isso não funcione.
```suggestion
            await this.page.click(this.$s.buttons.acceptCookie);
```